app/database.js

```diff
@@ -0,0 +1,54 @@
+/* file: database.js
+ * authors: H. Bouakaz, S. de Vliet, S. de Jong & E. Werkema
+ * date: 13/3/2016
+ * version 1.1
+ *
+ * Description:
+ * Module for loading the database. This file contains all the functions that
+ * update or retrieve information from the database. The dependancy for this
+ * module is sqlite.
+ */
+
+define(['sqlite'], function (sqlite) {
+	// Check if SQL.js has been loaded through AMD
+	var sql;
+	if (typeof sqlite !== 'object') {
+		document.body.style.backgroundColor = 'red';
+		alert("Failed to require sql.js through AMD");
```

Comment:
Is there a chance of this happening? If it does happen, present the user with some error page, or something like that.